### PR TITLE
Cherry-pick #19300 to 7.x: Update to go-libaudit v2.0.0

### LINKED ISSUE #14055: [Auditbeat] Add syscall names to to violated-seccomp-policy events
**Describe the enhancement:**

(Tested on Auditbeat 6.5.4)

when `event.action` is `violated-seccomp-policy`, there's `auditd.data.syscall` which is a number, but I want a name.

**Describe a specific use case for the enhancement or feature:**

When trying to understand what kinds of syscalls are causing seccomp violations, it would be more useful to have these syscalls available by name.

Here's a sample chart which isn't really useful unless you have memorized the syscall -> name mappings, and I don''t have these memorized ;)

firefox_0IaUlbaj2B

cc @stiltz who may be interested in this

### PR DESCRIPTION
**Pull request by andrewkroh**
Cherry-pick of PR #19300 to 7.x branch. Original message: 

## What does this PR do?

This updates the go-libaudit library to v2.0.0. Some of the updates since the last release were already
incorporated into Beats.

https://github.com/elastic/go-libaudit/releases/tag/v2.0.0

Fixes #14055 (seccomp field enrichments)
Fixes a few minor typos and spelling issues in auditd event.category and event.action values.

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -170,6 +170,10 @@ field. You can revert this change by configuring tags for the module and omittin
 - system/socket: Fix dataset using 100% CPU and becoming unresponsive in some scenarios. {pull}19033[19033] {pull}19764[19764]
 - system/socket: Fixed tracking of long-running connections. {pull}19033[19033]
 - system/package: Fix librpm loading on Fedora 31/32. {pull}NNNN[NNNN]
+- file_integrity: Create fsnotify watcher only when starting file_integrity module {pull}19505[19505]
```
Comment (adriansr) [inline CHANGELOG.next.asciidoc:173]: does this one belong here?

Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]: I'm glad you spotted that. It's fixed now.

**Comment by andrewkroh** [comment]
I've been seeing this unusual error from the Metricbeat tests:

```
[2020-07-15T08:36:14.406Z] + mage goIntegTest
[2020-07-15T08:37:22.185Z] Generated fields.yml for metricbeat to /var/lib/jenkins/workspace/Beats_beats_PR-19908/src/github.com/elastic/beats/metricbeat/fields.yml
[2020-07-15T08:37:22.185Z] Error: no module auditd
script returned exit code 1
```

I tracked down some discussion of it to https://github.com/elastic/beats/pull/18592#issuecomment-633618743. In this case the MODULE=auditd but that's only an Auditbeat module. Similarly I saw this with a Filebeat module change that triggered Metricbeat to build and it failed for the same reason ([logs](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats/detail/7.x/55/pipeline/1547#step-1788-log-26)). // fyi: @jsoriano @v1v

**Comment by v1v** [comment]
@andrewkroh , that particular misbehaviour was theoretically solved with https://github.com/elastic/beats/pull/18777 and backported. I'll run some tests locally and see if i can reproduce the behaviour for this PR

**Comment by v1v** [comment]
I see what's going on:
- The pattern to search for should not be generic but specific per beat, in other words: https://github.com/elastic/beats/blob/4dcbde39492bdc3843034bba8db811c68cb44b97/Jenkinsfile#L39-L40

should use `<beatname>` rather than `[a-z0-9]+beat` to avoid injecting the MODULE env variable to the Integration Tests stage that's related to another module.

If I recall we used the generic pattern to be easy to handle all the cases, but it seems it's not 100% accurate.

I'll raise an issue with these details and work on the fix.

**Comment by andrewkroh** [comment]
Thanks @v1v !

**Pull request merged**